### PR TITLE
Alerting: Add FGAC to alerts tab in panel editor

### LINKED ISSUE #47591: Alerting: FGAC alerts tab in panels
Add FGAC to the Alerts tab in a panel.

### PR DESCRIPTION
**Pull request by nathanrodman**
**What this PR does / why we need it**:

Hides the Alerts tab in the panel editor if the user doesn't have rule read access and hides the create alert button if they don't have create permissions.

**Which issue(s) this PR fixes**:

Fixes #47591 

**Special notes for your reviewer**:

### CODE REVIEW

**Review by konrad147 — APPROVED** [review]
LGTM!

File: public/app/features/dashboard/components/PanelEditor/state/selectors.ts [hunk 1]
```diff
@@ -35,7 +37,9 @@ export const getPanelEditorTabs = memoizeOne((tab?: string, plugin?: PanelPlugin
   }
 
   if (
```
Comment (gillesdemey) [inline public/app/features/dashboard/components/PanelEditor/state/selectors.ts:39]: Suggestion: Can we split these into  a couple of variables? I think that would make it a bit easier to reason about the boolean logic.

```js
const { alertingEnabled, unifiedAlertingEnabled } = getConfig();
const hasReadPermissions = contextSrv.hasPermission(getRulesPermissions('grafana').read);

const isGraph = plugin.meta.id === 'graph';
const isTimeseries = plugin.meta.id === 'timeseries'

if (alertingEnabled || (unifiedAlertingEnabled && hasReadPermissions && isGraph) || isTimeseries) { ... }
```

**Review by gillesdemey — APPROVED** [review]
LGTM, just a small code related suggestion but not a blocker :v:

File: public/app/features/dashboard/components/PanelEditor/state/selectors.ts (hunk 1 above)
Comment (konrad147) [reply public/app/features/dashboard/components/PanelEditor/state/selectors.ts:39]: Good point. I'll update

**Pull request merged**